static/src/javascripts/projects/commercial/modules/dfp/ophan-tracking.js

```diff
@@ -173,10 +173,7 @@ define([
     }
 
     function reportTrackingData() {
-        if (config.tests.commercialClientLogging ||
-            config.tests.commercialHbSonobi ||
```

Comment:
@rich-nguyen Could have left this here, but didn't see the point. Let's just report all users that are in the logging group.